superset/assets/javascripts/explore/stores/getLanguage.js

```diff
@@ -0,0 +1,24 @@
+
+const $ = window.$ = require('jquery');
+
+export function getLanguage() {
+  function getLocale() {
+    const locale = $.ajax({
+      url: '/superset/rest/api/getLocale',
+      async: false,
+    });
+    return JSON.parse(locale.responseText).language;
+  }
+  switch (getLocale()) {
```

Comment:
remove the switch and getLocale and return straight `return JSON.parse(locale.responseText).language;`. The switch does not buy us anything and also you have a typo and return spanish instead of english.